src/lib/decorators.ts

```diff
@@ -224,6 +224,14 @@ export function query(selector: string) {
       enumerable: true,
       configurable: true,
     };
+    if (cache) {
+      const key = typeof name === 'symbol' ? Symbol() : `__${name}`;
+      descriptor.get = function(this: LitElement) {
```

Comment:
I think this PR should update the docs, and mention that `null` isn't cached.

And are you sure you want `null` not cached? What's the use case vs not caching?

Reply:
Updated docs. The value `null` is now cached. The first value is always cached.